docs/installation/github.md

```diff
@@ -0,0 +1,60 @@
+# Download from GitHub
+
+The recommended way to install ScubaGear is [from PSGallery](psgallery.md), but it can also be downloaded from from GitHub:
```

Comment:
More concise hyperlinking and removed duplicate "from"
```suggestion
The recommended way to install ScubaGear is from [PSGallery](psgallery.md), but it can also be downloaded from GitHub:
```